### ISSUE TITLE
Reuse Variable but with different attributes

### ISSUE THREAD

**Issue by Jerry-2017** [apache/incubator-mxnet#6611]
Sometimes we need to share weights in network. And Its' advised in mxnet to reuse variable by applyinng  one weight and referencing multiple times. That's OK when variable attributes like lr_mult wd_mult is fixed.
But what if we want to change the learning rate in different network structure but still share the same weight.

My solution is to manually call symbol.set_attr with different learning_rate before different module initialization, is that OK? do you have a simpler way. Is that OK to define multiple variables with same name in this case? Thanks.

**Comment by piiswrong** [comment]
You can use different symbols for different modules

**Comment by Jerry-2017** [comment]
Thanks , I will have a try.

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
